unit/EspmTest.cpp

```diff
@@ -101,6 +101,35 @@ TEST_CASE("Loads LeveledItem", "[espm]")
   REQUIRE(data.chanceNone == 90);
 }
 
+TEST_CASE("Loads Conditions", "[espm]")
+{
+  auto& br = l.GetBrowser();
+  espm::CompressedFieldsCache cache;
+
+  auto form = br.LookupById(0x200E8F4);
+  REQUIRE(form.rec->GetType() == "COBJ");
+
+  auto data = reinterpret_cast<const espm::COBJ*>(form.rec)->GetData(cache);
+  REQUIRE(data.editorId ==
+          std::string("DLC1TemperArmorVampireCuirassRoyalRed"));
+  REQUIRE(data.conditions.size() == 2);
+
+  const int maxFunctionIndex = 726;
+  for (int i = 0; i < data.conditions.size(); i++) {
```

Comment:
(!!!) Most important:

This test is too permissive.

Instead of iterating conditions, please check each individual condition.

like REQUIRE(data.conditions[0].GetOperator() == espm::CTDA::Operator::EqualTo)
etc